packages/ipa/env/run.sh

```diff
@@ -1,9 +1,9 @@
 #!/bin/bash
 
-PWD=`pwd`/`dirname ${BASH_SOURCE}`
+CWD=`pwd`/`dirname ${BASH_SOURCE}`
 
-npx coffee $PWD/ipa/start.coffee
+npx coffee $CWD/ipa/start.coffee
 lxc exec freeipa bash <<EOF
 cd /nikita/packages/ipa
-npm test
+npx mocha 'test/**/*.coffee'
```

Comment:
Any reason why you switch to `npx mocha`?

Reply:
Yes, permission error when rebuilding a package, because of attached folders:
```
EACCES: permission denied, open '/nikita/packages/ipa/lib/group/add_member.js'
```